analyses/molecular-subtyping-embryonal/01-samples-to-subset.Rmd

```diff
@@ -112,40 +151,65 @@ fusion_biospecimens <- c(
 
 ### Disease labels
 
-First subset to embryonal **tumors**, excluding any derived cell lines.
+First subset to solid **tumors**, excluding any derived cell lines.
 
 ```{r}
 embryonal_df <- histologies_df %>%
-  filter(str_detect(broad_histology, "Embryonal tumor"),
-         sample_type == "Tumor",
+  filter(sample_type == "Tumor",
          composition == "Solid Tissue")
 ```
 
-Drop medulloblastoma and ATRT samples using the `pathology_diagnosis` labels.
+Now filter to include and exclude using the `pathology_diagnosis` and `pathology_free_text_diagnosis` labels.
+
+First, we will create an object filtered to include only samples with exact matches for the `path_dx_terms` (which is currently "Supratentorial or Spinal Cord PNET") in the `pathology_diagnosis` column.
+
+```{r}
+pnet_df <- embryonal_df %>%
+  # Filter for exact match of the defined pathology diagnosis term in `path_dx_list$include_path_dx`
+  filter(pathology_diagnosis %in% path_dx_list$include_path_dx)
+```
+
+Now we will create an objected filtered to include any sample with "Other" in the `pathology_diagnosis` column of the metadata, and with the `free_text_dx_terms` in the `pathology_free_text_diagnosis` column.
 
 ```{r}
-embryonal_df <- embryonal_df %>%
-  filter(!(pathology_diagnosis %in% c("Medulloblastoma",
-                                   "Atypical Teratoid Rhabdoid Tumor (ATRT)")))
+pathology_free_text_df <- embryonal_df %>%
+  # Filter for exact matches of `pathology_diagnosis == "Other"` and defined pathology free text diagnosis terms in `free_text_dx_terms`
+  filter(pathology_diagnosis == "Other",
+         pathology_free_text_diagnosis %in% path_dx_list$include_free_text) 
 ```
 
-Is there anything in the `integrated_diagnosis` column we should be concerned about removing?
+Any sample with "Neuroblastoma" in the `pathology_diagnosis` column, `primary_site != "Other locations NOS"`, and `pathology_free_text_diagnosis` does not contain "peripheral".
 
 ```{r}
-embryonal_df %>%
-  group_by(integrated_diagnosis) %>%
-  count()
+neuroblastoma_df <- embryonal_df %>%
+  filter(
+    str_detect(str_to_lower(pathology_diagnosis), "neuroblastoma"),
```

Comment:
Including the `Ganglioneuroblastoma` samples such that they are designated as `CNS Embryonal, NOS` makes sense to me based on the discussion, but there's nothing in the comments here or the documentation to note that the `Ganglioneuroblastoma` samples are being captured. There are also instances of `Metastatic secondary tumors;Neuroblastoma` in `pathology_diagnosis`, so otherwise we'd have to split based on `;` (this strategy would have applied to LGAT I believe). I'm inclined to add a detailed comment here and update the documentation throughout the module to make the inclusion of `Ganglioneuroblastoma` samples more clear.